jsonschema2pojo-cli/src/main/java/org/jsonschema2pojo/cli/Arguments.java

```diff
@@ -131,6 +131,9 @@
 
     @Parameter(names = { "-da", "--disable-accessors" }, description = "Whether to omit getter/setter methods and create public fields instead.")
     private boolean disableAccessors = false;
+    
+    @Parameter(names = { "-tv", "--target-version" }, description = "Target version for generated source files.")
```

Comment:
This is the best way to describe this setting.  Make all descriptions like this one.